README.md

```diff
@@ -3,13 +3,12 @@
 [![Issues](https://img.shields.io/github/issues/zeroeightysix/kami.svg)](https://github.com/zeroeightysix/kami/issues)
 [![Discord](https://img.shields.io/badge/chat-on%20discord-brightgreen.svg)](http://discord.gg/9hvwgeg)
 
-## A minecraft utility mod for anarchy servers.
+## A Minecraft utility mod for anarchy servers.
 
-Development has slowed down considerably.
+Forge 1.12.2 (currently `master`) is in slowed development. 
+Fabric 1.14.4 (currently `fabric-1.14`) is in active development. 
```

Comment:
Instead of making a README tailored to both branches, I'd much rather have it say that **this branch** is in slowed development, but people can look at the `fabric-1.14` branch instead for the newest updates.